### ISSUE TITLE
UI bug: no way to scroll to line of an issue being viewed

### ISSUE THREAD

**Issue by golubev** [reviewninja/review.ninja#1012]
In the issues list ("Review Status" tab) each issue has a quick link that scrolls user to the proper line of code:

![image](https://cloud.githubusercontent.com/assets/951364/11539599/911faabc-9930-11e5-83c3-70056955815d.png)
(http://joxi.ru/Dr8yn9OhkZbaOm)

But I was confused after I had clicked on an issue - there was no way to ask to scroll me to the proper line:

![image](https://cloud.githubusercontent.com/assets/951364/11539700/2d6a4f8a-9931-11e5-93c0-38c04c8e3a7b.png)
(http://joxi.ru/YmEakp0sZbXP4m)

I am really feeling a lack of a such feature.

**Comment by dfarr** [comment]
I see two possibilities here:
1) auto scroll when you click on an issue (pro: simple, con: automatically takes over scrolling)
2) provide link on details page as well

@golubev which would you prefer?

**Comment by golubev** [comment]
Hi @dfarr!

IMHO, auto scroll will be sometimes quite annoing. E.g. when I'm looking through the list of issues and don't want to see the code right now, or don't want to loose current scroll position.

I prefer a link.

**Comment by dfarr** [comment]
I like this link too :+1:, okay will take a look to figure something out

**Comment by golubev** [comment]
Thanks @dfarr!

**Comment by golubev** [comment]
Really lack this feature now, when I started looking at issues that were started on a diff that is outdated now.

I see that now when you open a PR-issue in ReviewNinja the diff is shown at a state it was in when the PR-issue was created. Really cool logic! :+1: 

But when I want to scroll to the code line the issue was started for I have no way to do it. I can't use the link shown in the first image above (the link in the PR-issues list), because the diff is outdated. And when I open the PR-issue - there is no link there at all... 

It comes that there is no way at all to navigate to a code line when a PR-issue was added to a line in a diff that is outdated now.
